packages/algob/src/internal/deployer.ts

```diff
@@ -147,6 +151,36 @@ class DeployerBasicMode {
     }
   }
 
+  /**
+   * Returns SSC checkpoint key using application index,
+   * returns undefined if it doesn't exist
+   * @param index Application index
+   */
+  getSSCCPKeyFromId (index: number): string | undefined {
```

Comment:
this is very cryptic name. Let's try to think about something better. `GetAppIndexCheckpoint`?

Reply:
this returns checkpoint key from given Id,
So maybe  getSSCKeyfromindex or getStatefulKeyFromIndex ?

Reply:
how about `getAppCheckpointKeyFromIndex` ?

Reply:
OK